inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -373,11 +374,20 @@ def prepare_files(obj, eng):
     if not obj.files:
         return
 
+    def _define_filename(paper, file_name):
+        """Define the filename in according to the article source"""
```

Comment:
There's no need to add docstrings to internal helpers: you're their only user, so they don't need documentation.